jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -536,32 +536,32 @@ properties:
             fits_keyword: OPMODE
             blend_table: True
           gwa_xtilt:
-            title: Grating Y tilt angle relative to mirror
+            title: Grating wheel tilt along instrmnt model Y axis
```

Comment:
Spelling of `instrmnt`: Is there a reason for the abbreviation?

Reply:
Yes. You can only fit a max of 47 characters in a FITS keyword comment before it starts getting truncated (runs off the end of the 80 char record).